charts/voltdb/templates/headless-svc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
should a headless service always be created?

Reply:
Yes, this is needed for voltdb cluster formation